src/fragments/lib/project-setup/flutter/upgrade-guide/upgrade-guide.mdx

```diff
@@ -1,328 +1,173 @@
-With the release of v1 Amplify Flutter now supports Web and Desktop for Auth, API, Analytics, and Storage use cases. Developers can build cross-platform Flutter apps with Amplify that target iOS, Android, Web, and Desktop (macOS, Windows, Linux) using a single codebase.
+Amplify Flutter v2 has several changes that may require migration when upgrading from v1.
 
-We have re-written our libraries in Dart. In some places, we have made breaking changes to improve ergonomics or enable features that had been missing from the v0 implementations.
+## Auth
 
-## Prerequisites
+#### Time based OTP Exception Handling
 
-- A Flutter application targeting Flutter SDK >= 3.3.0 with Amplify libraries integrated
+In Amplify Flutter v1, `Amplify.Auth.verifyTotpSetup()` throws an `EnableSoftwareTokenMfaException` if the provided code was incorrect. In Amplify Flutter v2 a `CodeMismatchException` is thrown.
```

Comment:
nit: First sentence uses active voice and second sentence uses passive voice. Let's use active in both.